openreview/venue/venue.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, client, venue_id, support_user):
         self.senior_area_chair_identity_readers = []
         self.automatic_reviewer_assignment = False
         self.decision_heading_map = {}
-        self.publication_chair = None
+        self.publication_chair = []
```

Comment:
can you use `self.publication_chairs` instead?

Reply:
why do you need this variable here?

Reply:
I think the most important place I use it is in the post decision to decide if they should be added as readers or not. I can try removing it but I would have to pass this as a parameter in a couple of places I believe

Reply:
But you need to the group id only, right?

Not the email addresses?

I agree we should have a setting to indicate the venue has publication chairs and it can be stored in the venue.content object, I think saving the publication_chairs_id field is enough.